Packs/CaseManagement-Generic/pack_metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "CaseManagement-Generic",
+    "description": "Case Management - Beta\n\nBuilt by the Cortex Customer Success Team to provide quick and valuable deployment of XSOAR for the main use of Case Management",
+    "support": "community",
+    "currentVersion": "0.0.5",
```

Comment:
```suggestion
    "currentVersion": "1.0.0",
```

Reply:
I'd like this to be published as a beta so not major release yet, is that ok?

Reply:
@omercnet I've deleted the `beta` tag you've added since it's not a valid one.
I've added the `beta` field with `true` value to the `pack_metadata` file, this to indicate that the pack is of beta version.
The `beta` field appears in the UI only in the following image:
![image](https://user-images.githubusercontent.com/53565845/91853825-71048980-ec6b-11ea-904c-83cdb1a068b5.png)
Is it ok from your end or another thing is needed to be shown to the user?
BTW, the pack metadata version should be `1.0.0`